test/conformance/enqueue/urEnqueueMemUnmap.cpp

```diff
@@ -2,45 +2,74 @@
 // Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
 // See LICENSE.TXT
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include "helpers.h"
 #include <uur/fixtures.h>
 
-struct urEnqueueMemUnmapTest : public uur::urMemBufferQueueTest {
+struct urEnqueueMemUnmapTestWithParam
+    : uur::urMemBufferQueueTestWithParam<uur::mem_buffer_test_parameters_t> {
     void SetUp() override {
-        UUR_RETURN_ON_FATAL_FAILURE(uur::urMemBufferQueueTest::SetUp());
+        UUR_RETURN_ON_FATAL_FAILURE(
+            uur::urMemBufferQueueTestWithParam<
+                uur::mem_buffer_test_parameters_t>::SetUp());
         ASSERT_SUCCESS(urEnqueueMemBufferMap(
             queue, buffer, true, UR_MAP_FLAG_READ | UR_MAP_FLAG_WRITE, 0, size,
             0, nullptr, nullptr, (void **)&map));
     };
 
-    void TearDown() override { uur::urMemBufferQueueTest::TearDown(); }
+    void TearDown() override {
+        uur::urMemBufferQueueTestWithParam<
+            uur::mem_buffer_test_parameters_t>::TearDown();
+    }
 
     uint32_t *map = nullptr;
 };
-UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urEnqueueMemUnmapTest);
 
-TEST_P(urEnqueueMemUnmapTest, Success) {
+static std::vector<uur::mem_buffer_test_parameters_t> test_parameters{
```

Comment:
since this same set of params gets used a few times could we move it into the header? as like a common buffer ops parameter set

Reply:
Good shout, have moved them into `helpers.h`. Thanks!